tests/e2e-leg-1/autoscale-by-pod/setup-vdb/base/setup-vdb.yaml

```diff
@@ -36,3 +36,4 @@ spec:
   imagePullSecrets: []
   volumes: []
   volumeMounts: []
+  encryptSpreadComm: disabled
```

Comment:
Why didn't you do the same for other e2e tests?

Reply:
I think encrypting spread channel will not affect other e2e tests. And since "enable" is the default behavior now, we can test the default behavior in other e2e tests. 

It turns out the admintool e2e tests are using old Vertica version(12.0.3) that does not support spread channel encryption without a db restart. Do you think should we update the Vertica version for the admintool e2e tests or should I disable spread channel encryption for the admintool e2e tests?